plugins/woocommerce/src/Blocks/BlockTypes/ProductMeta.php

```diff
@@ -7,7 +7,7 @@
 /**
  * ProductMeta class.
  */
-class ProductMeta extends AbstractBlock {
+class ProductMeta extends AtomicBlock {
```

Comment:
Given that `AtomicBlock` is a concept that we want to get rid of https://github.com/woocommerce/woocommerce/issues/42646, I would suggest applying these changes:

```
	/**
	 * Get the editor style handle for this block type.
	 *
	 * @return null
	 */
	protected function get_block_type_editor_style() {
		return null;
	}

	/**
	 * Get the frontend style handle for this block type.
	 *
	 * @return null
	 */
	protected function get_block_type_style() {
		return null;
	}


	/**
	 * Get the editor script data for this block type.
	 *
	 * @param string $key Data to get, or default to everything.
	 * @return null
	 */
	protected function get_block_type_editor_script( $key = null ) {
		return null;
	}
```
instead to extend the `AtomicBlock` class. Hopefully, we will remove it soon.


What do you think?

Reply:
👍